jetty-home/src/main/resources/modules/logging-noop.mod

```diff
@@ -1,23 +1,20 @@
-DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
 
 [description]
-Provides SLF4J API.  Requires a slf4j implementation (eg slf4j-simple-impl)
-otherwise a noop implementation is used.
+Configure logging to use SLF4J No-Op Implementation
```

Comment:
Somehow this is not working?
```shell
[1787] java -jar ../start.jar --add-to-start=server
INFO  : server          initialized in ${jetty.base}/start.d/server.ini
INFO  : logging-jetty   transitively enabled
INFO  : resources       transitively enabled
INFO  : threadpool      transitively enabled, ini template available with --add-to-start=threadpool
INFO  : bytebufferpool  transitively enabled, ini template available with --add-to-start=bytebufferpool
INFO  : Base directory was modified

[1788] java -jar ../start.jar 
2020-03-09 18:01:38.862:WARN :oejx.XmlConfiguration:main: Ignored arg: <Arg name="threadpool"><Ref refid="threadPool"/></Arg>
2020-03-09 18:01:38.898:INFO :oejs.Server:main: jetty-10.0.0-SNAPSHOT; built: 2020-03-09T16:23:20.158Z; git: 171a960fd39f35d0588853198dd90310c76193c2; jvm 13.0.2+8
2020-03-09 18:01:38.945:INFO :oejs.Server:main: Started Server@1d371b2d{STARTING}[10.0.0-SNAPSHOT,sto=5000] @477ms
^C
[1789] java -jar ../start.jar --add-to-start=logging-noop

ALERT: There are enabled module(s) with licenses.
The following 1 module(s):
 + contains software not provided by the Eclipse Foundation!
 + contains software not covered by the Eclipse Public License!
 + has not been audited for compliance with its license

 Module: logging-noop
  + SLF4J is distributed under the MIT License.
  + Copyright (c) 2004-2013 QOS.ch
  + All rights reserved.
  + Permission is hereby granted, free  of charge, to any person obtaining
  + a  copy  of this  software  and  associated  documentation files  (the
  + "Software"), to  deal in  the Software without  restriction, including
  + without limitation  the rights to  use, copy, modify,  merge, publish,
  + distribute,  sublicense, and/or sell  copies of  the Software,  and to
  + permit persons to whom the Software  is furnished to do so, subject to
  + the following conditions:
  + The  above  copyright  notice  and  this permission  notice  shall  be
  + included in all copies or substantial portions of the Software.
  + THE  SOFTWARE IS  PROVIDED  "AS  IS", WITHOUT  WARRANTY  OF ANY  KIND,
  + EXPRESS OR  IMPLIED, INCLUDING  BUT NOT LIMITED  TO THE  WARRANTIES OF
  + MERCHANTABILITY,    FITNESS    FOR    A   PARTICULAR    PURPOSE    AND
  + NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
  + LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
  + OF CONTRACT, TORT OR OTHERWISE,  ARISING FROM, OUT OF OR IN CONNECTION
  + WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

Proceed (y/N)? y
INFO  : logging-noop    initialized in ${jetty.base}/start.d/logging-noop.ini
INFO  : Base directory was modified

[1790] java -jar ../start.jar 
2020-03-09 18:01:58.722:WARN :oejx.XmlConfiguration:main: Ignored arg: <Arg name="threadpool"><Ref refid="threadPool"/></Arg>
2020-03-09 18:01:58.752:INFO :oejs.Server:main: jetty-10.0.0-SNAPSHOT; built: 2020-03-09T16:23:20.158Z; git: 171a960fd39f35d0588853198dd90310c76193c2; jvm 13.0.2+8
2020-03-09 18:01:58.791:INFO :oejs.Server:main: Started Server@5891e32e{STARTING}[10.0.0-SNAPSHOT,sto=5000] @451ms
^C
[1791] java -jar ../start.jar --dry-run
/opt/jdk-13.0.2/bin/java -Djava.io.tmpdir=/tmp -Djetty.home=/home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home -Djetty.base=/home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/test -cp /home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/lib/logging/slf4j-api-2.0.0-alpha1.jar:/home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/lib/jetty-servlet-api-4.0.3.jar:/home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/lib/jetty-http-10.0.0-SNAPSHOT.jar:/home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/lib/jetty-server-10.0.0-SNAPSHOT.jar:/home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/lib/jetty-xml-10.0.0-SNAPSHOT.jar:/home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/lib/jetty-util-10.0.0-SNAPSHOT.jar:/home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/lib/jetty-io-10.0.0-SNAPSHOT.jar org.eclipse.jetty.xml.XmlConfiguration java.version=13.0.2 java.version.major=13 java.version.micro=2 java.version.minor=0 java.version.platform=13 jetty.base=/home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/test jetty.base.uri=file:///home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/test jetty.home=/home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home jetty.home.uri=file:///home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home jetty.webapp.addSystemClasses=org.slf4j. slf4j.version=2.0.0-alpha1 /home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/etc/jetty-bytebufferpool.xml /home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/etc/jetty-threadpool.xml /home/gregw/src/jetty-10.0.x/jetty-home/target/jetty-home/etc/jetty.xml

```

Reply:
still not working?   No idea how it is finding our implementation as it doesn't appear on the dry run classpath?